pilz_trajectory_generation/test/unittest_trajectory_generator_circ.cpp

```diff
@@ -116,10 +121,6 @@ void TrajectoryGeneratorCIRCTest::SetUp()
 void TrajectoryGeneratorCIRCTest::checkCircResult(const planning_interface::MotionPlanRequest& req,
```

Comment:
This check function is nearly 100 lines long. Maybe it is advisable to put some of the checks done in this function in sub-functions with proper names instead of comments. Maybe it is even advisable to make some of the sub-functions into proper Gtest-Assert functions.